### PR TITLE
completed html-forms exercise, avallon smith 

### PR DESCRIPTION
**Pull request by avallonsmith**
animation of html-forms

Recorded this when I was still under the wrong folder, fixed it afterwards though!

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+<body>
+
+  <h1>Coding Form!</h1>
+
+  <label for="email">Email</label>
+  <div><input type="text" id="email" name="email"></div>
+  <br />
+
+  <form>
```
Comment (Zhiwutian) [inline html-forms/index.html:16]: there should only be one form tag on the page, and it should wrap all of the inputs

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <div>Why Is Coding Cool?</div>
```
Comment (Zhiwutian) [inline html-forms/index.html:17]: make sure to create a label for this input

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    <select name="languages">
+      <option value="Because it is interesting!">Because it is interesting!</option>
+      <option value="Because it is fun!">Because it is fun!</option>
+      <option value="Because it is powerful!">Because it is powerful!</option>
+    </select>
+  </form>
+  <br />
+
+  <form>
+    <div>Choose Your Favorite Language:<br />
```
Comment (Zhiwutian) [inline html-forms/index.html:27]: should be a label here as well

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      <input type="radio" name="language" value="HTML" checked="checked" />HTML
+      <input type="radio" name="language" value="CSS" />CSS
+      <input type="radio" name="language" value="Javascript" />Javascript
+    </div>
+  </form>
+  <br />
+
+  <form>
+    <div>Would you like to sign up for more information?<br />
```
Comment (Zhiwutian) [inline html-forms/index.html:36]: should be a label here

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+      <input type="checkbox" name="yes" value="Sign Me Up!" checked="checked" /> Sign Me Up!
+    </div>
+  </form>
+  <br />
+
+  <form>
+    <div>Any Additional Comments:<br />
```
Comment (Zhiwutian) [inline html-forms/index.html:43]: should be a label here as well

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
nice initial submission. Make sure to make updates based on the comments, and to remove the `br` tags. If an element needs to be broken onto a new line, divs are the way to go!

**Comment by avallonsmith** [comment]
image
image


Oh goodness, I am so sorry for such silly mistakes, guess I got a little ahead of myself haha. Hopefully, this is now fixed!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**